src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -92,13 +102,37 @@ export const SueReportLocation = ({
         locations={locations}
         mapCenterPosition={mapCenterPosition}
         mapStyle={styles.map}
+        isMyLocationButtonVisible
+        onMyLocationButtonPress={() =>
+          Alert.alert(texts.sue.report.alerts.hint, texts.sue.report.alerts.myLocation, [
+            {
+              text: texts.sue.report.alerts.no,
+              onPress: () => null
+            },
+            {
+              text: texts.sue.report.alerts.yes,
+              onPress: () => {
+                const location = position || lastKnownPosition;
+
+                if (location) {
+                  setUpdatedRegion(true);
+                  setSelectedPosition(location.coords);
+                  reverseGeocode(location.coords);
+                }
+              }
+            }
+          ])
+        }
         onMapPress={({ nativeEvent }) => {
-          setSelectedPosition({
-            ...nativeEvent.coordinate
-          });
-
+          setUpdatedRegion(true);
```

Comment:
I think `setUpdatedRegion(true)` is also unnecessary here.

Reply:
did you try it? we don't want to center to a region when launching the screen

Reply:
Yes, I tried, and you're right, it doesn't work as desired.